README.md

```diff
@@ -370,6 +370,7 @@ For documentation on the latest development code see the [documentation index][d
 - [SplunkMetric](/plugins/serializers/splunkmetric)
 - [Carbon2](/plugins/serializers/carbon2)
 - [Wavefront](/plugins/serializers/wavefront)
+- [MessagePack](/plugins/serializers/msgpack)
```

Comment:
Can you please sort alphabetically here!?

Reply:
Ack.

Reply:
Thanks for also cleaning up the other plugins as well! :-)